lib/engine/action/place_token.rb

```diff
@@ -7,19 +7,21 @@ module Action
     class PlaceToken < Base
       attr_reader :city, :slot, :token
 
-      def initialize(entity, city:, slot:, token_type: nil)
+      def initialize(entity, city:, slot:, tokener: nil, token_type: nil)
```

Comment:
why do you need tokener? can't you just infer it when the company lays the token?

Reply:
`Action` doesn't have access to the game outside of `self.h_to_args(h, game)`, passing `tokener` seemed less intrusive than giving `Action` full access to the game